build.gradle

```diff
@@ -318,26 +218,31 @@ task pushDockerImage {
     }
 }
 
-task removeOldDockerContainer(type: Exec) {
-    commandLine "bash", "-c",
-            "for c in `docker ps -qaf name=$dockerContainerName`; do docker rm -f \$c; done"
+task initNakadi(type: Exec) {
+    commandLine "bash", "-c", "docker-compose up -d --build"
+    doLast {
+        println "Waiting for Nakadi to start up"
+        long start = System.currentTimeMillis()
+        waitForNakadi()
+        println "Nakadi is fully started in " + (System.currentTimeMillis() - start) + " ms"
+    }
 }
 
-task startDockerContainer(type: Exec) {
-    commandLine "bash", "-c", "docker run -e \"NAKADI_OAUTH2_MODE=OFF\" -e \"NAKADI_FEATURETOGGLE_DEFAULT=true\" " +
-            "--net container:$storagesContainerName --name $dockerContainerName -i -t -d $dockerImageTag"
-    dependsOn buildDockerImage, removeOldDockerContainer, startStoragesInDockerBound
+task startNakadi(type: GradleBuild) {
+    tasks = ['clean', 'bootRepackage', 'initNakadi']
```

Comment:
Yes, exactly. In the initNakadi task we have docker compose started with option --build.